### ISSUE TITLE
Can not open second file with space in filename

### ISSUE THREAD

**Issue by getreu** [helix-editor/helix#2252]
### Summary

After `:vs` one can correctly select files with a space in the filename with the cursor.
Instead of opening that file, `hx` splits the selection wrongly using a space delimiter and tries to open the words individually.

### Reproduction Steps

I tried this:

1. `hx`

2. `:vs hello world.md`

I expected this to happen:

The file `hello world.md` opens.

Instead, this happened:

`hx` tries to open `hello` und `world.md`.

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux Debian 11

### Terminal Emulator

xfce4-terminal

### Helix Version

helix 22.05-dev (a8cb4668)

**Comment by the-mikedavis** [comment]
Wrapping the file name in quotes ("hello world.md") or using a backslash before the space to escape it (hello\\ world.md) only open the intended file.

I think it would make sense to have the file auto-complete insert the backslash version when completing.

**Comment by kirawi** [comment]
For me, it still seems to open two new splits. Maybe you tried `hx "hello world.txt"` (which works) and not `:vs "hello world.txt"`?

**Comment by the-mikedavis** [comment]
Ah you caught me, I was using `:o` 😅

I just gave it a try with `:vs` though and it only opens the one split. I don't see the same behavior with it making multiple new splits 🤔

**Comment by getreu** [comment]
When you surround the filename (manually) with double quotes, the file opens as expected. Still, this is very confusing (and not ergonomic) because: when you scroll with `[tab]` through the file list, you expect `[Enter]` to open the file you just selected.
Instead it opens 2 or 3 empty tabs.

**Comment by getreu** [comment]
Probably the easiest way to fix this is:
* When selecting a filename with tab, it should be quoted automatically (e.g. `"hello world.md"`) and only then copied into the bottom editor line (e.g. `vs: "hello world.md"`).

**Comment by getreu** [comment]
https://github.com/helix-editor/helix/pull/1231 is not a fix for this issue, but seems related.

**Comment by woojiq** [comment]
Fixed by #4316
